test/invalid/invalid-missing-general-usage/README.md

```diff
@@ -0,0 +1,563 @@
+# table-of-contents-no-heading [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
+
+a valid origami readme
```

Comment:
Could we change this to state it is invalid instead?

```suggestion
an invalid origami readme
```

Reply:
💯